lib/xclarity_client.rb

```diff
@@ -1,6 +1,8 @@
 require 'xclarity_client/version'
+require 'utils/logger'
 
 module XClarityClient
+  LOG = (XClarityClient::XClarityLogger.new).freeze
```

Comment:
I don't think that this should be called LOG all caps.  I think something like _log would be appropriate. This pattern is similar to what manageIQ has done.